cli/app.go

```diff
@@ -379,6 +380,44 @@ var app = &cli.App{
 						},
 					},
 				},
+				{
+					Name:      "database",
+					Usage:     "interact with a MongoDB Atlas Data Federation instance",
+					UsageText: "viam data database [other options]",
+					Subcommands: []*cli.Command{
+						{
+							Name:      "configure",
+							Usage:     "configures a database user for Viam's MongoDB Atlas Data Federation instance",
```

Comment:
nit, but this makes it sound like it's one instance for all of Viam. Maybe `"configures a database user for the Viam org's MongoDB Atlas Data Federation instance"`

Reply:
Updated! Will update in API too